components/automate-ui/src/app/pages/+compliance/+scanner/containers/scanner/scanner.component.html

```diff
@@ -8,10 +8,10 @@
 
       <ul class="nav-tabs">
         <li class="nav-tab" routerLink="/compliance/scan-jobs/jobs" routerLinkActive="active">
-          <span *ngIf="jobsCountLoaded">{{jobsCount$ | async | number}}</span> Scan jobs 
+          <span *ngIf="jobsCountLoaded">{{(jobsCount$ | async | number) === '0' ? '' : (jobsCount$ | async | number)}}</span> Scan Jobs 
```

Comment:
No need to stringify in the test:
```suggestion
          <span *ngIf="jobsCountLoaded">{{(jobsCount$ | async) === 0 ? '' : (jobsCount$ | async | number)}}</span> Scan Jobs 
```
Same change in L14 below.

Reply:
@msorens , I have added review comment changes.